configure.ac

```diff
@@ -141,6 +142,9 @@ AC_CONFIG_FILES([scripts/missing-multichars.sh],
 AC_CONFIG_FILES([scripts/regresstest-pipespec.sh],
                 [chmod +x scripts/regresstest-pipespec.sh])
 
+# Enable .drb support if divvun-runtime is available
+AM_CONDITIONAL([HAVE_DIVVUN_RUNTIME], [test x$DIVVUN_RUNTIME != xfalse])
```

Comment:
I think you want this test in template-lang-und and applied to all langs? C.f. https://builds.giellalt.org/pipelines/lang-liv/builds/7 line 221